src/modules/Dashboard/components/TakeNowModal/LabeledDropdown/LabeledUserDropdown.tsx

```diff
@@ -19,6 +25,7 @@ export const LabeledUserDropdown = ({
   onChange,
   handleSearch,
   debounce,
+  disabled,
```

Comment:
Why the explicit disabled prop here?

Reply:
It's used to affect the colour of the `popupIcon` SVG. 🦅